api/Selection.json

```
@@ -586,6 +619,39 @@
             "deprecated": false
           }
         },
+        "accepts_node_in_any_tree": {
```

All of these updates 
```suggestion
        "accepts_nodes_in_shadow_tree": {
```